app/src/main/java/fr/free/nrw/commons/category/GridViewAdapter.java

```diff
@@ -99,8 +99,8 @@ public View getView(int position, View convertView, ViewGroup parent) {
      */
     private void setAuthorView(Media item, TextView author) {
         if (!TextUtils.isEmpty(item.getCreator())) {
+            author.setVisibility(View.VISIBLE);
             String uploadedByTemplate = getContext().getString(R.string.image_uploaded_by);
-
             String uploadedBy = String.format(Locale.getDefault(), uploadedByTemplate, item.getCreator());
             author.setText(uploadedBy);
```

Comment:
I know this isn't your code but can you fix this for me? 

Just use `author.setText(context.getString(R.string.image_uploaded_by, item.getCreator()))`

Reply:
Sure